api/hub/src/api.ts

```diff
@@ -402,6 +402,35 @@ api.post('/moderation/moderators/new', async (ctx: koa.Context, next: () => Prom
   await next();
 });
 
+/**
+ * Update an existing moderator.
+ */
+api.post('/moderation/moderators/:user', async (ctx: koa.Context, next: () => Promise<any>) => {
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/AKASHAorg/akasha-world-framework/pullRequest?prid=8098753)
```suggestion
api.post('/moderation/moderators/:user', async (ctx: koa.Context, next: () => Promise<unknown>) => {
```